### DISCUSSION TITLE
jpg file conversion failing

### DISCUSSION THREAD

**Discussion by amannagarro01** [ImageMagick/ImageMagick#7831] [General]
Hi,
Thumbnails of attached jpg file are not getting generated using ImageMagick, while I am able to open this file properly in AdobePhotoshop. 
![LPL-Wood_Wood_1](https://github.com/user-attachments/assets/3ba05079-d107-494e-b0a0-00179e4ecf08)

Can someone help here to see why this image thumbnails doesn't generates using ImageMagick commands.

**Comment by snibgo** [comment]
What version of IM? What command did you use? Did you get any error messages?

I can't download the JPG file. Firefox says "... cannot be displayed because it contains errors."

**Comment by dceeee** [comment]
IM reports
magick: Not a JPEG file: starts with 0x2d 0x2d

The start of the file is
----------------------------220084305834635446877917
Content-Disposition: form-data; name="file"; filename="LPL-Wood_Wood_1.jpg"
Content-Type: image/jpeg
ÿØÿà  JFIF

ÿØÿà starts at byte 159 not zero. Delete the junk in a hex editor and it works or open/save with Irfanview or Photoshop

**Comment by snibgo** [comment]
That explains the problem. IM is more strict than some other software about reading only files that comply with standards, for security reasons.

**Reply by amannagarro01** [reply]
Thanks @snibgo @dceeee for your response.
